openstack/compute/v2/extensions/aggregates/requests.go

```diff
@@ -39,3 +41,11 @@ func Create(client *gophercloud.ServiceClient, opts CreateOpts) (r CreateResult)
 	})
 	return
 }
+
+// Delete makes a request against the API to delete an aggregate.
+func Delete(client *gophercloud.ServiceClient, aggregateID int) (r DeleteResult) {
+	v := strconv.Itoa(aggregateID)
+	_, r.Err = client.Delete(aggregatesDeleteURL(client, v), &gophercloud.RequestOpts{
+		OkCodes: []int{200}})
```

Comment:
This is a very minor nit and I apologize, but we should make this look like the other OkCodes:

```go
_, r.Err = client.Delete(aggregatesDeleteURL(client, v), &gophercloud.RequestOpts{
        OkCodes: []int{200},
})

Reply:
Wow, I'm surprised that `go fmt` didn't fix it.

Reply:
Yeah, I was surprised that both `fmt` _and_ `vet` didn't flag it. Thank you for fixing this :)